src/how-to/install/demo-wiab.md

```diff
@@ -0,0 +1,379 @@
+- Can be skipped with `skip_asset_host=true`
+
+### 11. Container Seeding
+
+- Imports [seed-offline-containerd.yml](https://github.com/wireapp/wire-server-deploy/blob/master/ansible/seed-offline-containerd.yml) to seed containers in K8s cluster nodes
+- It will seed the docker images shipped for the wire related helm charts in the minikube k8s nodes
+- Can be skipped with `skip_setup_offline_seed=true`
+
+### 12. Wire Secrets Creation
+
+- Imports [wire_secrets.yml](https://github.com/wireapp/wire-server-deploy/blob/master/ansible/wiab-demo/wire_secrets.yml) to create required secrets for wire helm charts
+- Only runs if both `skip_wire_secrets` and `skip_helm_install` are false
+
+### 13. Helm Chart Installation
+
+- Imports [helm_install.yml](https://github.com/wireapp/wire-server-deploy/blob/master/ansible/wiab-demo/helm_install.yml) to deploy Wire components using Helm
+- These charts can be configured in [host.yml](https://github.com/wireapp/wire-server-deploy/blob/master/ansible/inventory/demo/host.yml)
+- Can be skipped with `skip_helm_install=true`
+
+### 14. Temporary Cleanup
+
+- Locates all temporary SSH key directories created during deployment
+- Lists and removes these directories
+
+## SSH Proxy Configuration
+
+The deployment uses an SSH proxy mechanism to access:
+1. Kubernetes nodes within the Minikube cluster
+2. The asset host for resource distribution
+
+SSH proxying is configured with:
+    - Dynamic discovery of SSH key paths (uses `ansible_ssh_private_key_file` if defined)
+    - StrictHostKeyChecking disabled for convenience
+    - UserKnownHostsFile set to /dev/null to prevent host key verification issues
+
+## General Tips
+
+### Ansible run selective tasks
+- You can use ^skip_ variables as environment variables to control the execution flow of the playbook. If these variables are passed, they will skip specific groups of tasks as explained in the [Deployment Flow](#deployment-flow) section. By default, if no variables are passed, all tasks will run in sequence.
+
+  In case of timeouts or other failures, you can skip tasks that have already been completed by passing the appropriate flags. For example, if the Wire artifact download task fails due to a timeout or disk space issue, you can skip the preceding tasks by using the following command:
+```bash
+ansible-playbook -i ansible/inventory/demo/host.yml ansible/wiab-demo/deploy_wiab.yml -e "skip_verify_dns=true skip_install_pkgs=true skip_minikube=true"
+```
+  This command will skip the DNS verification, package installation, and Minikube cluster configuration tasks, allowing you to resume the playbook from the Wire artifact download task.
+
+### Iptables behaviour
+
+- All the iptables rules are not persisted after reboots, but they can be regenerated by running the entire pipeline or restored from the `/home/ansible_user/wire-iptables-rules/rules_post_wire.v4` directory. Optionally, we can skip everything else when generating the rules again.
```

Comment:
restored from the... so, just CDing to that directory restores iptables rules, eh? :)